akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -250,6 +250,11 @@ object Source {
    * The actor will be stopped when the stream is completed, failed or canceled from downstream,
    * i.e. you can watch it to get notified when that happens.
    *
+   * An async boundary is automatically created after the returned actorRef source, so downstream
+   * operations will not occur in the same thread as the Actor ingesting messages. If you select too
```

Comment:
I absolutely agree that it should be documented that it will drop messages if the buffer of the `actorRef` source is overflown and appreciate that you bring this up. I was only objecting about the documentation of the internal async boundary and any internal buffers related to that.

What about something like this, directly after "... there is no space available in the buffer.":

Note that there is no backpressure signal from the source to the sender of the messages. That means that if you send messages faster than the stream is consuming them the buffer will fill up and eventually overflow, resulting in dropped messages.